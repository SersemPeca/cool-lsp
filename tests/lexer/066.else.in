else
Else
eLse
elSe
elsE
ELse
eLSe
elSE
ElSe
eLsE
ElsE
ELSe
eLSE
ElSE
ELsE
ELSE
