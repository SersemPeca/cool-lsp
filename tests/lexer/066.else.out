#1 ELSE
#2 ELSE
#3 ELSE
#4 ELSE
#5 ELSE
#6 ELSE
#7 ELSE
#8 ELSE
#9 ELSE
#10 ELSE
#11 ELSE
#12 ELSE
#13 ELSE
#14 ELSE
#15 ELSE
#16 ELSE
